pubsub/pubsub.go

```diff
@@ -423,7 +423,15 @@ func (s *Subscription) Receive(ctx context.Context) (_ *Message, err error) {
 func (s *Subscription) getNextBatch(ctx context.Context, nMessages int) ([]*Message, error) {
 	var msgs []*driver.Message
 	for len(msgs) == 0 {
-		err := retry.Call(ctx, gax.Backoff{}, s.driver.IsRetryable, func() error {
+		// Allow unrecoverable errors from acks to be detected and reported.
```

Comment:
What would happen if we allowed `getNextBatch` to return an empty slice instead? Wouldn't we just loop in `Receive`?

E.g., try commenting out your new code, and also deleting the `for len(msgs) == 0` loop in this function, does it wor?

Reply:
It does appear to work. Good idea.